src/main/java/sim/coffee/KitchenQueueGUI.java

```diff
@@ -1,26 +1,34 @@
 package sim.coffee;
 
+import java.awt.GridLayout;
 import java.util.LinkedList;
 
 import javax.swing.BorderFactory;
+import javax.swing.JPanel;
 import javax.swing.JScrollPane;
 import javax.swing.JTextArea;
 import javax.swing.border.Border;
 
-public class KitchenQueueGUI extends JTextArea implements Observer{
+public class KitchenQueueGUI extends JPanel implements Observer{
 	
 	SharedQueue kitchenQueue;
+	JTextArea queue;
 	
 	public KitchenQueueGUI(SharedQueue kitchenQueue) {
 		this.kitchenQueue = kitchenQueue;
 		kitchenQueue.registerObserver(this);
+		setLayout(new GridLayout(1,1));
```

Comment:
This may be a stupid question, but why the GridLayout here? 😅

Probably a good idea to have a comment if there's an unobvious reason for it

Reply:
To get the JPanel take up all the available space in the GUI, doesn't require setting a specific size so works nicely with resizing as well 😅 
I'll add in a comment.

Reply:
Ah nice 👍 Yeah I recall having to do similar for some of stage 1 so that makes sense to me. Definitely a prime candidate for a "why" comment.